### ISSUE TITLE
Pokemon X not Work

### ISSUE THREAD

**Issue by ToniBC** [libretro/citra#90]
Hello, I have tried the game of Pokemon X / Y and they do not work, after putting the name, the screen goes black.

Everything tested on Retroarch x64 under Windows 10, everything updated to the latest nightly and updated cores.

The game also has graphical bugs that are not in the standalone versions. In this version, everything works without problems.

Here are some examples of the graphic errors on the boy's face.

![Pokemon X (Europe) (En,Ja,Fr,De,Es,It,Ko)-210805-200720](https://user-images.githubusercontent.com/45365478/128400430-960e4e25-b1e5-4108-a8ee-0ed25c09ea8c.png)
![Pokemon X (Europe) (En,Ja,Fr,De,Es,It,Ko)-210805-200749](https://user-images.githubusercontent.com/45365478/128400442-412a8068-3047-4977-b8f6-83cc79ddb892.png)

Basically the problem is that the core is not updated, it is very backward or old since all these problems and others are solved in the standalone versions.
